ui.frontend/html/src/constants.js

```diff
@@ -1,8 +1,8 @@
 // Here the different views are specified
 const views = {
   MAIN: 'main-page',
-  ADD_CONTACT: 'add-contact',
-  START_SESSION_WIZARD: 'start-session-wizard',
+  ADD_CONTACT_PAGE: 'add-contact-page',
+  SHARE_PROJECT_PAGE: 'share-project-page',
   CONFIGURATION_WIZARD: 'configuration-page',
```

Comment:
Same as in `App.jsx`. Remaining usage of 'wizard', which was now generally replaced with 'page' as far as I can tell.